.circleci/config.yml

```diff
@@ -10,93 +10,81 @@
 # DOCKER_PASS
 
 version: 2
-
-# This is not part of circleci, but it lets us not repeat ourselves.
-STUFF:
-  python_image: &python_image
-    image: python:3.8-alpine
-
-  # it's useful to not build branches that contain slashes,
-  # if it is later decided to map branches to docker tags.
-  main_filters: &main_filters
-    branches:
-      ignore:
-        - /\//
-
 jobs:
-  lint:
-    working_directory: /app
+  full:
     docker:
-      - *python_image
-    steps:
-      - checkout
-      - run: apk add gcc linux-headers libc-dev
-      - run: pip install black==19.10b0 && pip install flake8==3.7.9
-      - run: black --diff .
-      - run: flake8 .
-  test:
-    working_directory: /app
-    docker:
-      - *python_image
+      - image: circleci/python
+        user: root
     steps:
+      - setup_remote_docker
       - checkout
-      - run: |
-          /app/docker/install-deps.sh \
-            /app/requirements.txt
-      - run: pip --no-cache install .
       - run:
-          name: run tests
+          name: Create version.json
           command: |
-            mkdir test-reports
-            pytest --junitxml=test-reports/junit.xml
-          environment:
-            # this is needed for pytest to run
-            PYTHONPATH: /app
-            # this lets you watch the build progress in the web UI
-            PYTHONUNBUFFERED: 1
-
-      - store_test_results:
-          path: test-reports
+            echo "
+              {
+                \"commit\": \"$CIRCLE_SHA1\",
+                \"version\": \"$CIRCLE_SHA1\",
+                \"source\": \"https://github.com/mozilla-conduit/lando-ui\",
+                \"build\": \"$CIRCLE_BUILD_URL\"
+              }
+            " > version.json
+            mkdir /tmp/artifacts && cp version.json /tmp/artifacts/version.js
 
+      - run:
+          name: Build the image
+          command: |
+            docker build --pull \
+              -f docker/Dockerfile-prod\
```

Comment:
What does the backslash at the end here do?

Reply:
Oh, I see. I wasn't sure if it was somehow part of the path.
Would you mind adding a space between `Dockerfile-prod` and `\`?